src/ray/common/ray_config_def.h

```diff
@@ -116,6 +116,10 @@ RAY_CONFIG(int64_t, max_direct_call_object_size, 100 * 1024)
 // limit in Ray to avoid crashing with many small inlined task arguments.
 RAY_CONFIG(int64_t, max_grpc_message_size, 100 * 1024 * 1024)
 
+// Retry timeout for trying to create a gRPC server. Only applies if the number
+// of retries is non zero.
+RAY_CONFIG(int64_t, grpc_server_retry_timeout_milliseconds, 100)
```

Comment:
I think this should be longer by default, maybe something like 1 or 2 seconds. The port conflict scenario should be rare

Reply:
Okay!